__tests__/app.test.js

```diff
@@ -209,6 +209,63 @@ describe("POST /api/reviews/:review_id/comments", () => {
     })
 })
 
+describe("PATCH /api/reviews/:review_id", () => {
+    test("201: Object updated when given the right parameters", () => {
```

Comment:
technically this should be a 200 rather than a 201 as no new resource has been created, just updated.